tests/functional/r/regression_02/regression_8736.txt

```diff
@@ -0,0 +1 @@
+comparison-of-constants:3:7:3:37::"Comparison between constants: '𐀀 == ' has a constant value":HIGH
```

Comment:
```suggestion
comparison-of-constants:3:7:3:37::"Comparison between constants: '"\U00010000" == "\ud800\udc00"' has a constant value":HIGH
```
Imo we should aim for this, but I have an intuition that the ast module internal might be involved.

Reply:
Yes, but I don't think we can. `self.out` is `utf-8` encoded so we always need to encode/escape to `utf-8`.